torch/testing/_internal/common_utils.py

```diff
@@ -55,6 +57,13 @@
 import torch.backends.mkl
 from enum import Enum
 
+try:
+    sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../.."))
+    from tools.stats_utils.s3_stat_parser import (get_S3_bucket_readonly, HAVE_BOTO3, Report, Version2Report)
+except ImportError:
+    print("Unable to import s3_stat_parser from tools. Running without S3 stats...")
+    HAVE_BOTO3 = False
```

Comment:
I am not sure this would work well here, I only tried this when the script is `__main__`

Reply:
Yea, this doesn't work because the __file__ is the one in site_packages.